src/GitTools.Core/Extensions/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace GitTools
+{
+    using System.Text;
+
+    public static class StringBuilderExtensions
+    {
+        //[StringFormatMethod("format")]
```

Comment:
I think we should add the r# annotations nuget to make the usage of these light up?